exercises/all-your-base/all_your_base_test.rb

```diff
@@ -1,8 +1,7 @@
-#!/usr/bin/env ruby
 require 'minitest/autorun'
 require_relative 'all_your_base'
 
-# Common test data version: 1.0.0 39de23b
+# Common test data version: 1.1.0 c4d8d95
```

Comment:
The tests should not be updated other than to remove the shebang line.
You may need to checkout this file and update it manually.